docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -465,3 +467,116 @@ The following overrides are available:
+
+### Prepare OpenShift clusters for MCS
+
+Red Hat OpenShift Container Platform uses the Advanced Cluster Management for Kubernetes (RHACM) and its Submariner addon to enable MCS. At a very high level this involves following steps:
+
+1. Create a management cluster and install RHACM on it. For details, see [Installing Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing).
+1. Provision the OpenShift clusters which will be connected together.  
+   Ensure that the CIDRs mentioned in the cluster configuration file at `networking.clusterNetwork`, `networking.serviceNetwork`, and `networking.machineNetwork` are non-overlapping across the multiple clusters. You can find more details about these options in provider specific sections under the [OpenShift Container Platform installation overview](https://docs.openshift.com/container-platform/4.11/installing/index.html), look for section "Installing a cluster on [provider name] with customizations".
+1. Import the clusters into RHACM as a cluster set, and install Submariner addon on them. For more information, see the section from [1.1.2. Configuring Submariner](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/add-ons/add-ons-overview#configuring-submariner).  
+   Note down the cluster names from the cluster set, these will be used during the cloud provider setup in YugabyteDB Anywhere.
+
+### Prepare Kubernetes clusters for Istio mulit-cluster
+
+Istio service mesh can span across multiple clusters which allows us to configure MCS. It supports different topologies and network configurations. Follow [Install Multicluster](https://istio.io/latest/docs/setup/install/multicluster/) to get started with the multi-cluster Istio setup.
+
+The Istio configuration for each cluster should have following options:
+
+```yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_DNS_CAPTURE: "true"
+        ISTIO_META_DNS_AUTO_ALLOCATE: "true"
+  # rest of the configuration…
+```
+Refer the Istio setup section from [Multi-Region YugabyteDB Deployments on Kubernetes with Istio](https://www.yugabyte.com/blog/multi-region-yugabytedb-deployments-on-kubernetes-with-istio/) for a more step by step guide along with explanation of the options being used.
+
+### Configure the cloud provider for MCS
+
+Once you have the cluster setup ready, you can follow [Configure the Kubernetes cloud provider](#). And refer to this section for region and zone configuration requried for multi-region universes.
```

Comment:
Link to come?

Reply:
Want it to jump to the top of the page in case someone has directly reading the MCS sections, hence the `#`.